### ISSUE TITLE
facebook.GraphAPIError

### ISSUE THREAD

**Issue by bulletproofsysadmin** [wooyek/flask-social-blueprint#14]
When testing this I get the following when trying to perform basic authentication to facebook.  GraphAPIError: An active access token must be used to query information about the current user.

**Comment by bulletproofsysadmin** [comment]
BTW I'm getting an access_token back from core.login.  I'm just not certain that the model is handling it properly.

**Comment by toddetzel** [comment]
I submitted pull request #15 which addresses this bug. Looks like it was a compatibility-breaking change in the latest Facebook Python SDK which mangled the querystring such that the access token wasn't being passed down.

**Comment by Arti3DPlayer** [comment]
same issue:
An active access token must be used to query information about the current user

**Comment by Arti3DPlayer** [comment]
Can you update pip version please ?

**Comment by wooyek** [comment]
Yes, I'll do that, give a me a minute please. I want to update the test env first… and it's Christmas ;)

**Comment by toddetzel** [comment]
Thanks, Janusz! Merry Christmas to you. :)

**Comment by wooyek** [comment]
Please check if it's ok now. 

PS. There are issues with Github provider on GAE #18, but they seem to work ok locally.
